docs/reference/ml/anomaly-detection/apis/put-job.asciidoc

```diff
@@ -105,6 +105,21 @@ include::{es-ref-dir}/ml/ml-shared.asciidoc[tag=custom-rules]
 (array)
 include::{es-ref-dir}/ml/ml-shared.asciidoc[tag=custom-rules-actions]
 
+//Begin analysis_config.detectors.custom_rules.params
+`params`:::
+(object)
+include::{es-ref-dir}/ml/ml-shared.asciidoc[tag=custom-rules-params]
++
+.Properties of `params`
+[%collapsible%open]
+=======
+[[force-time-shift-params]]
```

Comment:
This is a leftover of another possible solution to present the docs. After its removal, docs build should be okay.
```suggestion
```